### ISSUE TITLE
Simplify chrome execution debugging

### ISSUE THREAD

**Issue by dmail** [jsenv/core#54]
**Current steps to debug**

- needs a file starting an exploring server to execute any file
- set a port both in exploring server and jsenv-chrome debug configuration in launch.json
- start the exploring server
- use the jsenv-chrome debug configuration

Steps before jsenv-chrome can be done once per project.
Server must be started or jsenv-chrome will fail.
After that jsenv-chrome debug configuration can be used.

**Simplification using launchChromium**

We could use launchChromium to start a chromium browser in debug mode -> it's already possible.
Then we could instruct vscode to connect to that chromium to debug it -> have to write our own vscode debugger or make https://github.com/microsoft/vscode-chrome-debug compatible with that concept.

**Simplification using prelaunch task**

It would prevent having to start the server but blocked by https://github.com/microsoft/vscode/issues/6209.
The solution where vscode connected to the chromium would be cleaner.

**Comment by dmail** [comment]
Certainly won't do, exploring server and a regular browser is better
